main.cpp

```diff
@@ -79,42 +101,79 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+int answerToEverything(bool knowsQuestion, float timeToCalculate = 7.5f, int answer = 42)
+{
+  ignoreUnused(knowsQuestion, timeToCalculate, answer);
+  return {};
+}
 /*
  2)
  */
-
+double convertCurrency(char convertFrom, char convertTo, float amount, bool roundUp = false)
+{
+  ignoreUnused(convertFrom, convertTo, amount, roundUp);
+  return {};
+}
 /*
  3)
  */
-
+void delayLoudspeakers(int maxDistance, bool isOutdoor = true, float temperature = 20.0f)
+{
+  ignoreUnused(maxDistance, isOutdoor, temperature);
+}
 /*
  4)
  */
-
+bool needsPhantomPower(char transducerType = 'c')
+{
+  ignoreUnused(transducerType);
+  return {};
+}
 /*
  5)
  */
-
+int getSurfaceArea(int width = 0, int height = 1)
+{
+  ignoreUnused(width, height);
+  return {};
+}
 /*
  6)
  */
-
+void waveLength(float lambda, float frequency, double speedOfSound = 343.6)
+{
+  ignoreUnused(lambda, frequency, speedOfSound);  
+}

```

Comment:
Fix the indentation.
It should be set to '4 spaces'.
This is currently using an indentation of '2 spaces'
- [ ] revise